### ISSUE TITLE
Bitcoin ABC Uses same registry entries as Bitcoin core

### ISSUE THREAD

**Issue by PigletBoy** [Bitcoin-ABC/bitcoin-abc#40]

**Comment by snavsenv** [comment]
A M A T E U R S

**Comment by PigletBoy** [comment]
I agree, but it is not correct

**Comment by darrenkis** [comment]
LOL

**Comment by xgiovio** [comment]
yes. same problem. temporary fix, set the data dir with "C:\Program Files\Bitcoin\bitcoin-qt.exe" -datadir="<my bitcoin data directory>"

**Comment by ohiTuna** [comment]
> yes. same problem. temporary fix, set the data dir with "C:\Program Files\Bitcoin\bitcoin-qt.exe" -datadir=""        

Would advise caution if doing this. I did -datadir="x:\mybitcoincoredatadir" and have managed to send bitcoin (not bcc) to unwanted addresses. Will post issue specifics shortly.

**Comment by xgiovio** [comment]
I did not have any problem

**Comment by bacoboy** [comment]
Dupe of #32 or vice versa?

**Comment by schancel** [comment]
#32 is a slightly different issue

**Comment by GSPP** [comment]
Another duplicate of this is https://github.com/Bitcoin-ABC/bitcoin-abc/issues/234.

Needing to run both Core and ABC is a reality for many users in the community. On Windows the only way to do that that seems safe enough is using virtual machines. This is cumbersome to manage and has disk space and memory usage overheads.

Ideally, simply installing the next version of Bitcoin ABC should show a message box *"Do you want to migrate your data directory and registry configuration to the recommended locations?"*. If yes is clicked this would upgrade everything and make the machine clean for a subsequent Core installation.

In my opinion this issue should be prioritized as a matter of usability.

**Comment by GSPP** [comment]
I just tried a fresh installation. The following paths are used:

C:\Program Files\Bitcoin-abc
C:\Users\%User%\AppData\Roaming\Bitcoin
HKEY_LOCAL_MACHINE\SOFTWARE\Bitcoin ABC (64-bit)
